srcopsmetrics/bot_knowledge.py

```diff
@@ -51,18 +59,22 @@ def analyse_projects(
         project_path = path.joinpath("./" + github_repo.full_name)
         check_directory(project_path)
 
-        allowed_entities = [e.value for e in EntityTypeEnum]
+        allowed_entities = get_all_entities()
```

Comment:
why not Enums?

Reply:
I've done the dynamic imports from the entities folder. WDYT?

Reply:
Sounds good :)